### ISSUE TITLE
Example Code to only search given data folder

### ISSUE THREAD

**Issue by getsean** [techleadhd/chatgpt-retrieval#20]
How would we remove the ability for OpenAI answers to come through. The YouTube video wasn't super clear on this.

**Comment by JohnOstrowick** [comment]
Agreed. I gave it pdfs with very clear specific content and it could not answer questions about them except from data which was already online prior to 2021, which suggests that its' just pulling answers from headquarters instead of the local pdf repo.

**Comment by JohnOstrowick** [comment]
I'd like an "ingest" option to explicitly tell it to ingest the PDFs provided, or, a progressbar or something showing that it is doing precisely that.
